armi/reactor/assemblies.py

```diff
@@ -713,8 +713,13 @@ def setBlockMesh(self, blockMesh, conserveMassFlag=False, adjustList=None):
         belowFuelColumn = True
 
         if self[-1].p.topIndex == 0:
-            # this appears to not have been initialized, so initialize it
-            self.makeAxialSnapList(refMesh=blockMesh)
+            runLog.warning(
```

Comment:
we initialize this in all assemblies, including blueprints assemblies in processLoading now, so every assembly should have this except for the ones we want to ignore expansion on.